docs: update rate limit policy (#9405)
zitadel/zitadel: docs/docs/legal/policies/rate-limit-policy.md

> custom_edit_url: null

Author: @stebenz do you know what this field is about?
Do we have to change something elsewere?

:::info Raising limits
We understand that there are certain scenarios where your users access ZITADEL from shared IP Addresses.
For example if you use a corporate proxy or Network Address Translation NAT.
Please [get in touch](https://zitadel.com/contact) with us to discuss your requirements, and we'll find a solution.
:::

## What rate limits do apply

> For ZITADEL Cloud, we have a rate limiting rule for user interface paths (login, register and reset features, console) and for API paths each.

Reviewer: ```suggestion
For ZITADEL Cloud, we have a dedicated rate limits for the user interfaces (login, register, console,...) and the APIs.
```

Rate limits are implemented with the following rules:

| Path                 | Description                                                             | Rate Limiting                        | One Minute Banning                    |
|----------------------|-------------------------------------------------------------------------|--------------------------------------|---------------------------------------|
| /ui/\*               | Global Login, Register and Reset Limit                                  | 10 requests per second over a minute | 15 requests per second over 3 minutes |

> | All other paths      | All gRPC-, REST and OAuth APIs as well as the ZITADEL Customer Portal   | 50 requests per second over a minute | 50 requests per second over 3 minutes |

Reviewer: ```suggestion
| All other paths      | All gRPC-, REST and OAuth APIs  | 50 requests per second over a minute | 50 requests per second over 3 minutes |
```

Reviewer: I would not mentioned the portal ATM
